usecases.rst

```diff
@@ -14,3 +14,13 @@ Data Integration
 Transform disparate and disconnected data-sources -- such as 3rd-party APIs, legacy infrastructure, or databases into streams of typed, structured records -- which can be composed together. For instance, you could set up a workflow in the cloud which pipes the Twitter Ads API into your data-lake (and even do some modelling in Python in-transit) in under 5 minutes.
 
 
+Software Lifecycle
+******************
+
+NStack provides best practices from software engineering and end-to-end software life-cycle management to the data science process, including,
+
+* **sharing and reuse** - build and deploy individual model modules to your cloud that can reused, imported, and utilised by other members, either within your team or globally
+* **reproducibility** - build models with guaranteed versions of system and language dependencies, that can then be shared with confidence that your code is identical everywhere
+* **versioning** - modules a have a immutable, globally unique, version - so when importing to use in a workflow, you can be sure the API won't change
```

Comment:
API but also the code and artifacts "are the right ones". Maybe mention updating too and how this plays in